dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -0,0 +1,69 @@
+from dissect.target.helpers.fsutil import TargetPath, decompress_and_readlines
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+from datetime import datetime
+import re
+
+AuthLogRecord = TargetRecordDescriptor(
+    "linux/log/auth",
+    [
+        ("datetime", "ts"),
+        ("string", "msg"),
+    ],
+)
+
+
+class AuthPlugin(Plugin):
+    def check_compatible(self):
+        return self.target.fs.path("/var/log/auth.log").exists() or self.target.fs.path("/var/log/secure").exists()
+
+    @export(record=[AuthLogRecord])
+    def securelog(self):
+        return self.authlog()
+
+    @export(record=[AuthLogRecord])
+    def authlog(self):
+        """
+        Yields AuthLogRecords from /var/log/auth.log* and /var/log/secure*
+        """
+        auth_files: [TargetPath] = list(self.target.fs.path("/var/log/").glob("auth.log*")) + list(
+            self.target.fs.path("/var/log/").glob("secure*")
+        )
+
+        RE_TS = r"^[A-Za-z]{3}\s*[0-9]{1,2}\s[0-9]{1,2}:[0-9]{2}:[0-9]{2}"
+        RE_TS_AND_HOSTNAME = RE_TS + r"\s\w+\s"
+
+        for auth_file in auth_files:
+
+            file_ctime = self.target.fs.get(str(auth_file)).stat().st_ctime
+            year_file_created = datetime.fromtimestamp(file_ctime).year
```

Comment:
Basically same comments in this loop regarding the bytes and datetime stuff as in #131 so I won't repeat here.